Calendar/BookingHelpers/ExcelWrite.ps1

```diff
@@ -0,0 +1,124 @@
+        Write-Host -ForegroundColor Red "Graph modules are not installed. Exiting..."
+        exit
+    }
+
+    Connect-MgGraph -Scopes "User.Read.All", "Bookings.Read.All" -NoWelcome
+
+    Write-Host -ForegroundColor Green "Exporting to Excel..."
+
+    #Compose File name formatted as: BookingsSummary_<IdentityName>_<yyyy-MM-dd>_<HHmm>.xlsx
+    $IdentityName = SplitIdentityFromEmail -email $identity
+    $DateStringToFileName = (Get-Date).ToString("yyyy-MM-dd")
+    $HourStringToFileName = Get-Date -Format "HHmm"
+    $path = "BookingsSummary_$($IdentityName)_$($DateStringToFileName)_$($HourStringToFileName).xlsx"
+    Remove-Item -Path $path -Force -EA Ignore
+
+    $ExcelParamsArray = GetExcelParams -path $path -tabName "Business"
+
+    Write-Host "Exporting Booking Business."
+    $BusinessData = GetGraphBookingBusiness -Identity $identity
+    $excel = $BusinessData | Export-Excel @ExcelParamsArray -PassThru
+    Export-Excel -ExcelPackage $excel -WorksheetName "Business" -MoveToStart
+
+    Write-Host "Exporting Bookings Page Settings."
+    $BusinessPage = GetGraphBookingBusinessPage -Identity $identity
+    $BusinessPage | Export-Excel -Path  $path -WorksheetName "Business" -StartRow 10 -Title "Page Settings" -TableStyle Medium3 -AutoSize
+
+    Write-Host "Exporting Booking Policy."
+    $BookingPolicy = GetGraphBookingBusinessBookingPolicy -Identity $identity
+    $BookingPolicy | Export-Excel -Path  $path -WorksheetName "Business" -BoldTopRow  -StartRow 10 -StartColumn 10 -Title "Scheduling Policy" -TableStyle Medium3 -AutoSize
+
+    Write-Host "Exporting Working Hours."
+    $WorkingHours = GetGraphBookingBusinessWorkingHours -Identity $identity
+    $WorkingHours | Export-Excel -Path  $path -WorksheetName "Business" -BoldTopRow  -StartRow 25  -Title "Working Hours" -TableStyle Medium24 -AutoSize
+
+    Write-Host "Exporting Staff."
+    $Staff = Get-GraphBookingsStaff -Identity $identity
+    $Staff | Export-Excel -Path  $path -WorksheetName "Staff" -TableStyle Medium3 -AutoFilter -FreezeTopRow -BoldTopRow -MoveToEnd -AutoSize -Title "Staff List"
+
+    Write-Host "Exporting Services."
+    $Services = Get-GraphBookingsServices -Identity $identity
+    $Services | Export-Excel -Path  $path -WorksheetName "Services" -TableStyle Medium3 -AutoFilter -FreezeTopRow -BoldTopRow -MoveToEnd -AutoSize -Title "Services"
+
+    Write-Host "Exporting Custom Questions."
+    #Get-MgBookingBusinessCustomQuestion -BookingBusinessId $identity | Export-Excel -Path  $path -WorksheetName "Custom Questions" -AutoFilter -FreezeTopRow -BoldTopRow -MoveToEnd
+    $CustomQuestions = Get-GraphBookingsCustomQuestions -Identity $identity
+    $CustomQuestions | Export-Excel -Path  $path -WorksheetName "Custom Questions" -TableStyle Medium3 -AutoFilter -FreezeTopRow -BoldTopRow -MoveToEnd -AutoSize -Title "Custom Questions"
+
+    Write-Host "Exporting MessageTrace for the past $($script:MessageTrackingDays) days."
```

Comment:
Do you want these to be Write-Host or Write-Verbose?
probably write-host, but just checking...

Reply:
Write-Host